src/rule-parser.js

```diff
@@ -141,27 +140,16 @@ class EventRequirement extends  Requirement {
   }
 
   conditionFulfilled (condition, matchedEvent) {
-    /*switch(condition) {
```

Comment:
Here's why this `switch` statement didn't work initially: The `switch` statement compares the value it receives within parentheses (in this case `condition`) with all the values returned by the expressions of the `case` statements. Our `case` statements return boolean values and we want to execute the `case` statement where the expression evaluates to `true`. This is why we need to pass `true` to the `switch` statement, not the `condition` variable itself.

Reply:
Oh man...this makes so much sense now. We couldn't wrap our heads around this. Thanks for clarification!